src/worker.js

```diff
@@ -105,6 +105,15 @@ Module['instantiateWasm'] = (info, receiveInstance) => {
 }
 #endif
 
+// Turn unhandled rejected promises into errors so that the main thread will be
+// notified about them.
+self.onunhandledrejection = (e) => {
+  if (typeof e.reason !== 'undefined') {
+    throw e.reason;
+  }
+  throw e;
```

Comment:
This can just be `throw e.reason ? e.reason : e`?

Reply:
Actually you do this these days I think: `throw e.reason ?? e`

Reply:
done.